internal/release/main.go

```diff
@@ -119,3 +121,27 @@ func generateFile(name, version string) error {
 	jsonEncoder.SetIndent("", "  ")
 	return jsonEncoder.Encode(downloadArchive)
 }
+
+func Builder() *cobra.Command {
+	opts := Opts{}
+	cmd := &cobra.Command{
```

Comment:
[non-blocking] using cobra for this is an overkill if you ask me, you could've used pflags with no need to bring cobra as well